docs-team/2016-12/december-22.md

```diff
@@ -0,0 +1,68 @@
+# 2016-12-22 Agenda
+
+### Previously
+
+- [x] DB: Talk with Gavin re: videos
+- [x] DB: will talk with Trek re: static page hosting, can use Github pages?
+
+Topics
+
+- DB: Help Wanted Update
+- DB: EmberMap, testing discussion, did any of us review?
+- DB: website storage
+- RM: [Learn page showcase](https://github.com/emberjs/website/pull/2722#issuecomment-266288860)
+- TJ: API doc updates
+
+# Action Items 
+
+- [ ] RM: CouchDB access for Ilya
+- [ ] TJ: PR meeting notes from last few weeks
+- [ ] TJ: Work with ryanto on test video
+- [ ] TJ: Reach out to Tom Dale about future API changes with modules, and questions about namespaces.
+
+# Minutes
+
+## Attendees
+
+- TJ: Todd Jordan @toddjordan
+- DB: David Baker @acorncom
+- RM: Ricardo Mendes @locks
+- GM: Gaurav Munjal @gaurav
+
+## Notes
+
+---
+
+### Netlify
+
+DB: Netlify accounts setup up.  RM mentions it was put in the core agenda.  Not sure what that means.  Things seem to be running smoothy and think we can turn it on.
+
+RM: We should wait to make the switch because the ember website also needs to change hosting because of github pages rate limiting.  nathan was going to reach out to github.  There may be some concerns about netlify.  Will talk with the core team tomorrow and propose Netlify.  Netlify dev in my working spaces, will reach out to him.
```

Comment:
Hello GitHub! We'd love to be able to leverage GitHub Pages for emberjs/emberjs.github.com (AKA emberjs.com) but we're limited by the number of requests that we would need to support. Is this a limit you'd be willing or able to work with us on? We're entirely comfortable with changing cache lifetimes and optimizing our process to make sure that we play nicely on your infrastructure.

What I've heard is that we are probably a factor of 10 over your current limits. Let us know what you think! Thanks!

Nathan Hammond
(Acting as an official emissary from the Ember community.)

---

Hello Nathan,

We <3 open source so are happy to work with great projects to meet their needs as best we can.

The limits are soft limits so your site won't suddenly go dark on request 100,001. We generally reach out to projects that are affecting our infrastructure in advance and help them, when possible, to reduce the impact.

Looking over your current usage patterns though, we don't see any major problems in the near future.

You can also help us out by taking a few steps on your end.

One is that you might want to consider putting CloudFlare in front of the site. Both to be able to put https on emberjs.com and also to leverage their CDN to bring down the traffic to the GitHub Pages servers.

A second one is to limit any hotlinking to the project releases by community members. i.e. don't use GitHub Pages as a CDN for sites other than emberjs.com.

It looks like you are already making good use of Github Releases for that though and I couldn't see any obvious cases of this with a browse of the site, so you should be good to go there.

If there are any potential issues you can think of from your end, or something specific you need from us please let us know.

Regards,
Daniel
GitHub Support